wren-ai-service/README.md

```diff
@@ -6,26 +6,89 @@ Please read the [documentation](https://docs.getwren.ai/oss/concept/wren_ai_serv
 
 ## Setup for Local Development
 
-### Environment Setup
+### Prerequisites
 
-- Python 3.12.\*, recommended to use [`pyenv`](https://github.com/pyenv/pyenv?tab=readme-ov-file#installation) to manage the Python versions
-- install `poetry` with version 1.8.3: `curl -sSL https://install.python-poetry.org | python3 - --version 1.8.3`
-- execute `poetry install` to install the dependencies
-- copy `.env.dev.example` file to `.env.dev` and fill in the environment variables
-- [for development] execute `poetry run pre-commit install` to install the pre-commit hooks and `poetry run pre-commit run --all-files` to run the pre-commit checks at the first time to check if everything is set up correctly
-- [for development] install [Just](https://github.com/casey/just?tab=readme-ov-file#packages)
-- [for development] to run the tests, execute `just test`
+1. **Python**: Install Python 3.12.\*
 
-### Start the service for development
+   - Recommended: Use [`pyenv`](https://github.com/pyenv/pyenv?tab=readme-ov-file#installation) to manage Python versions
 
-The following commands can quickly start the service for development:
+2. **Poetry**: Install Poetry 1.8.3
 
-- `just up` to start needed containers
-- `just start` to start the service 
-    - go to `http://WREN_AI_SERVICE_HOST:WREN_AI_SERVICE_PORT`(default is http://localhost:5556) to see the API 
-    documentation and try them.
-    - go to `http://WREN_UI_HOST:WREN_UI_PORT`(default is http://localhost:3000) to interact interact from the UI
-- `just down` to stop the needed containers
+   ```bash
+   curl -sSL https://install.python-poetry.org | python3 - --version 1.8.3
+   ```
+
+3. **Just**: Install [Just](https://github.com/casey/just?tab=readme-ov-file#packages) command runner
+
+### Step-by-Step Setup
+
+1. **Install Dependencies**:
+
+   ```bash
+   poetry install
+   ```
+
+2. **Generate Configuration Files**:
+
+   ```bash
+   just init true
```

Comment:
I suggest default should be true, since the command is used for local development